services/federator/src/Federator/Remote.hs

```diff
@@ -169,12 +169,12 @@ blessedCiphers =
     TLS.cipher_TLS13_AES128CCM_SHA256,
     TLS.cipher_TLS13_AES128GCM_SHA256,
     TLS.cipher_TLS13_AES256GCM_SHA384,
-    TLS.cipher_TLS13_CHACHA20POLY1305_SHA256,
+    -- TLS.cipher_TLS13_CHACHA20POLY1305_SHA256, -- not compliant to TR-02102-2
     -- For TLS 1.2 (copied from default nginx ingress config):
     TLS.cipher_ECDHE_ECDSA_AES256GCM_SHA384,
     TLS.cipher_ECDHE_RSA_AES256GCM_SHA384,
-    TLS.cipher_ECDHE_RSA_AES128GCM_SHA256,
     TLS.cipher_ECDHE_ECDSA_AES128GCM_SHA256,
-    TLS.cipher_ECDHE_ECDSA_CHACHA20POLY1305_SHA256,
-    TLS.cipher_ECDHE_RSA_CHACHA20POLY1305_SHA256
+    TLS.cipher_ECDHE_RSA_AES128GCM_SHA256
+    -- TLS.cipher_ECDHE_ECDSA_CHACHA20POLY1305_SHA256, -- not compliant to TR-02102-2
+    -- TLS.cipher_ECDHE_RSA_CHACHA20POLY1305_SHA256  -- not compliant to TR-02102-2
```

Comment:
Can you entirely remove the lines of code, rather than leaving them commented out?

Also, if this applies a general recommendation, could you link to the recommendation (perhaps this is about https://www.bsi.bund.de/SharedDocs/Downloads/EN/BSI/Publications/TechGuidelines/TG02102/BSI-TR-02102-2.html ?) in a comment above the `blessedCiphers` definition for future reference?